### PR TITLE
Show emoji flags for log lines with locales

### PR DESCRIPTION
**Pull request by Turbo87**
bildschirmfoto 2017-02-27 um 22 10 42

this is using:

- https://github.com/iarna/has-unicode to figure out if the terminal supports unicode (and possibly emojis)
- https://github.com/10xjs/locale-emoji to turn locale string to emoji

/cc @jasonmit

### CODE REVIEW

**Comment by jasonmit** [comment]
Love it 👍   Merge when ready

File: package.json [hunk 1]
```diff
@@ -72,10 +72,12 @@
     "ember-intl-relativeformat": "^2.4.0",
     "exists-sync": "0.0.3",
     "extend": "^3.0.0",
+    "has-unicode": "^2.0.1",
```
Comment (jasonmit) [inline package.json]: Should also update the yarn lockfile

Comment (Turbo87) [inline package.json]: I used yarn to install so lockfile update is apparently not necessary as some subdependency depends on it already. Likely npm via ember-cli...

**Comment by Turbo87** [comment]
Yeah, will need a bit of cross platform testing, but looks promising so far.

File: package.json (hunk 1 above)
Comment (jasonmit) [inline package.json]: 👍

**Comment by Turbo87** [comment]
just noticed that it's using the UK flag for en-us though... Something is not working as expected yet :thinking_face:

**Comment by Turbo87** [comment]
bildschirmfoto 2017-02-27 um 22 31 48

Looks like we need to pass the country part in uppercase. I'll file an issue to support lowercase too.

**Comment by stefanpenner** [comment]
🎉

**Comment by jasonmit** [comment]
@Turbo87 I kicked off another build since the last one hung.

**Pull request merged**